### ISSUE TITLE
Add conda forge infrastructure

### ISSUE THREAD

**Issue by tcmetzger** [earth-system-radiation/pyRTE-RRTMGP#24]
Update our existing conda recipe and build infrastructure to publish the package on conda forge (for Linux/Ubuntu x86 only, for now).

Goals:

- [x] On a Ubuntu 64-x86 system, the package is installable with `conda install pyrte_rrtmgp` (or equivalent) (or at least: our entry to conda forge is under review)
- [x] The installation instructions in the readme reflect the new option to install via conda forge

**Comment by tcmetzger** [comment]
PR on conda forge: https://github.com/conda-forge/staged-recipes/pull/25894

Currently on pause until an official release of RTE RRTMGP is available.

**Comment by RobertPincus** [comment]
[v1.8 of RTE+RRTMGP ](https://github.com/earth-system-radiation/rte-rrtmgp/releases/tag/v1.8) contains the kernel API C header files.

**Comment by tcmetzger** [comment]
From our 11/20 call:

- The data is different from the python code. we should have a separate conda package pyrte-data (just the scheme data). pyrte could then have that as a dependency. pyrte-data is a subset of the data in the fortran data release
- Also: don't really run the tests during conda build (we test thoroughly in our CI/CD pipeline in the repo), to avoid hitting the restrictions of the conda build vm.

**Comment by giancastro** [comment]
@tcmetzger , the tests will make sure the conda package will work. We need to pass the tests.

**Comment by tcmetzger** [comment]
@giancastro Somewhat related to this, it seems like we'll need to find a better way to download and store the test data, in cases where a user has installed the package from conda!

**Comment by tcmetzger** [comment]
Yes, that is what we'll do - part of this is to define where to store that data from the conda package so that it is accessible to the tests, etc. (as defined in https://github.com/earth-system-radiation/pyRTE-RRTMGP/issues/24#issuecomment-2489216828).

@RobertPincus @sehnem Could you add a comment here to define which data (i.e. which folders or files) from https://github.com/earth-system-radiation/rrtmgp-data should be included in the future `pyrte-data` package?

**Comment by tcmetzger** [comment]
From our 2024/12/04 discussion: 
We decided that the current implementation and function for downloading the data is sufficient for now. The only use case we want to cover is: after the user installs the conda package into their local environment, they want some way of verifying that the package works correctly. This could be some small "healthcheck" CLI command that downloads a small amount of data and runs a subset of the pytests. 

@brendancol and @giancastro to look into this in https://github.com/earth-system-radiation/pyRTE-RRTMGP/issues/73

**Comment by tcmetzger** [comment]
I'm closing this as done - the next step is in https://github.com/earth-system-radiation/pyRTE-RRTMGP/issues/62 (once the fortran package from https://github.com/earth-system-radiation/pyRTE-RRTMGP/issues/61 is available)
